jaws-journey-deploy/orb.yml

```diff
@@ -515,7 +515,7 @@ jobs:
                 account-url: AWS_ACCOUNT_URL
                 aws-access-key-id: AWS_ACCESS_KEY_ID
                 aws-secret-access-key: AWS_SECRET_ACCESS_KEY
-                dockerfile: ./<< parameters.serviceName >>/Dockerfile
+                dockerfile: Dockerfile
```

Comment:
why is this change required?

Reply:
Started failing at a later step [https://app.circleci.com/pipelines/github/ovotech/jaws-journey-reconcile-device-store/341/workflows/cb4223d0-7b91-4343-884c-19e9d59935a5/jobs/3381](https://app.circleci.com/pipelines/github/ovotech/jaws-journey-reconcile-device-store/341/workflows/cb4223d0-7b91-4343-884c-19e9d59935a5/jobs/3381) - seems that the dockerfile variable now gets appended to the path, trying to reduce duplication I suppose